contributors/guide/pull-requests.md

```diff
@@ -82,24 +82,26 @@ Here's the process the pull request goes through on its way from submission to m
 1. (Optional) Some reviewers prefer that you squash commits at this step
 1. Follow the bot suggestions to assign an OWNER who will add the `/approve` label to the pull request
 
-Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `/lgtm` and `/approved`, the pull request enters the final merge queue. The merge queue is needed to make sure no incompatible changes have been introduced by other pull requests since the tests were last run on your pull request.
+Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `/lgtm` and `/approved`, the pull request enters the final merge pool. The merge pool is needed to make sure no incompatible changes have been introduced by other pull requests since the tests were last run on your pull request.
```

Comment:
> all failures are commented as flakes

What does this mean?

Reply:
> What does this mean?

Not sure, it was part of the old docs. I've updated it. Ptal.